### ISSUE TITLE
Small Medical Update

### ISSUE THREAD

**Issue by JRBXX1** [studio-minus/ppg-bugs-and-requests#2932]
### Feature description

Add a ventilator to help someone breathe without lungs  - punctured lung etc
Add a brain activity monitor to diagnose brain death without inspect - brain death

### Feature purpose

because it will make the game more realistic, i know the cardiopulmonary bypass machine already breathes for the lungs but it would be cool to have one seprate

**Comment by ghost** [comment]
CPB is your ventilator, there's no practical reason for dividing an existing item

**Comment by JRBXX1** [comment]
yep. but doesn't it have to be running to ventilate? if a person just had a punctured lung but not a stopped heart, their BPM would be 70 the whole time with the CPB on; not allowing the heart to pump independently but using a machine. the reason I would say is realism/being realistic. most hospitals don't put patients on cardiopulmonary bypass just for ventilation. but they would put the patient on a ventilator, just saying.

**Comment by ATPGaming** [comment]
Here before DindinYT37, if he's even still here. Try your best not to multi-request, it makes it harder to accept features individually.
